torch/testing/_internal/custom_op_db.py

```diff
@@ -162,7 +162,7 @@ def numpy_nonzero_impl(x):
 def numpy_nonzero_abstract(x):
     ctx = torch._custom_op.impl.get_ctx()
     i0 = ctx.create_unbacked_symint()
-    shape = [x.dim(), i0]
+    shape = [i0, x.dim()]
```

Comment:
the tuple axis (which is being added) is dim length, so if you insert it at dim=1 it is at the end. You could resolve this either way but this was the minimum change.